src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.simplefileshare.simplefileshare.controller;
```

Comment:
Can we keep only one package either `controller` or `controllers` and move both the example controller and FileController there?

Reply:
okay.